.golangci.yml

```diff
@@ -1,21 +1,25 @@
 run:
   timeout: 240m
-  skip-dirs:
-  - pkg/complianceoperator/api
-  - scanner/updater/rhel        # TODO(ROX-21539): Remove when CSAF/VEX arrives
   build-tags:
   - integration
   - sql_integration
   modules-download-mode: readonly
+  go: "1.21"
 
 output:
-  format: "junit-xml:report.xml,colored-line-number"
+  formats:
+    - format: colored-line-number
+    - format: junit-xml
+      path: report.xml
 
 issues:
   max-issues-per-linter: 0
   max-same-issues: 0
   exclude-use-default: false
 
+  exclude-dirs:
+  - pkg/complianceoperator/api
+  - scanner/updater/rhel        # TODO(ROX-21539): Remove when CSAF/VEX arrives
```

Comment:
Should we add `generated/.*` here?

Reply:
I'm not quite sure we should, AFAIK golangci-lint helped us discover problems with cloning protos regarding the mutex, not sure whether this would have worked if generated was excluded - I'd just keep things as-is for now.